src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -181,6 +181,28 @@ class ActivityFeed extends React.Component<Props, State> {
         deleteComment(args);
     };
 
+    /**
+     * Callback for successful creation of a Task.
+     *
+     * @param {Task} task - API returned task
+     * @param {string} id - ID of the feed item to update with the new task data
+     * @return {void}
+     */
+    createTaskSuccessCallback(task: Task, id: string): void {
+        this.updateFeedItem(task, id);
+    }
+
+    /**
+     * Callback for error while creating a Task.
+     *
+     * @param {Error} error - Error thrown while creating the Task
+     * @param {string} id - ID of the feed item to update as no longer pending
+     * @return {void}
+     */
+    createTaskErrorCallback(error: Error, id: string): void {
+        console.log('Implement createTaskErrorCallback() via updateFeedItemPendingStatus()', error, id);
```

Comment:
console.error? Or call the default error handler that does logging

Reply:
using `console.error()` and updating for `createCommentErrorCallback`